docs/getting-started/sdk/secrets-manager/index.mdx

```diff
@@ -0,0 +1,20 @@
+# Secrets Manager
+
+This section contains development information for the Bitwarden Secrets Manager CLI, language
+wrappers, and integrations based on the SDK.
+
+## Web Client
+
+To start the web client, follow the
+[Web Vault setup instructions](../../clients/web-vault/index.mdx) and create an organization with
+Secrets Manager enabled. Then use the org switcher to go to Secrets Manager.
```

Comment:
⛏️ You sometimes use the shorthand "org".

Reply:
@differsthecat this was one of the pieces you wrote.  Do you have any issue with this change?

Reply:
No issue at all, thank you!